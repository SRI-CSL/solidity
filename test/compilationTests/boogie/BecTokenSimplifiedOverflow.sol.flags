--solver cvc4 --boogie-arith mod-overflow