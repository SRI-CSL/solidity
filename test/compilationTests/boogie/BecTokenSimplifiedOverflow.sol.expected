Verifying BecTokenSimplified::[constructor]: OK
Verifying BecTokenSimplified::balanceOf: OK
Verifying BecTokenSimplified::transfer: OK
Verifying BecTokenSimplified::batchTransfer: ERROR
 - Source BecTokenSimplifiedOverflow.sol, line 66, col 32: An overflow can occur before calling function
